py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/helpers.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
```

Comment:
all LGTM.

Reply:
i did move it into this dir so that we can access it via this module rather than lib.

i think we should make some naming decisions soon. part of this is my own ignorance on not remembering all of the different acronyms we're supposed to keep track of.